showcase/src/app/welcome/pages/reactive-form-errors/demo-reactive-form-errors-page.component.html

```diff
@@ -0,0 +1,163 @@
+<div class="container">
+	<h1 class="mat-display-1" translate>
+		SHOWCASE.NGX_FORM_ERRORS.TITLE
+	</h1>
+	<h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
+	<section class="stark-section">
+		<example-viewer
+			class="progress-indicator-example"
+			[extensions]="['HTML', 'SCSS', 'TS']"
+			filesPath="reactive-forms-with-ngx-form-errors/reactive-forms-with-ngx-form-errors"
```

Comment:
Can you rename the folder of the examples to `reactive-form-errors` to keep consistency?